desktop/components/react/utils/render_layout.js

```diff
@@ -0,0 +1,36 @@
+import renderTemplate from 'desktop/components/react/utils/render_template'
```

Comment:
Actually....let's say we were just passed in a function that returns a string. Would `renderToString` simply succeed? If so, maybe we don't have anything to worry about. If not, then I think we just need to catch the error that causes:

```
const blockOutput = block(locals)
try {
  html = renderToString(blockOutput)
} catch (ex) {
  if (/* whatever exception would be thrown if `block` returned raw HTML */ && isString(blockOutput)) {
    html = blockOutput
  } else {
    // fail loudly to devs, gracefully to user
 }
}
```